app/graphql/mutations/profile/create.rb

```diff
@@ -18,26 +18,38 @@ class Create < BaseMutation
       field :profile, Types::Profile, null: false
 
       def resolve(args = {})
-        profile = ::Profile.new(new_profile_options(args))
-        profile.save!
-        profile.update_rules(ref_ids: args[:selected_rule_ref_ids])
+        policy = ::Policy.new(new_policy_options(args)).fill_from(
+          profile: ::Profile.find(args[:clone_from_profile_id])
+        )
+        profile = ::Profile.new(new_profile_options(args)).fill_from_parent
+
+        Policy.transaction do
+          policy.save!
+          profile.update!(policy: policy)
+          profile.update_rules(ref_ids: args[:selected_rule_ref_ids])
+        end
+
         { profile: profile }
       end
 
       private
 
-      def new_profile_options(args)
+      def new_policy_options(args)
         {
           account_id: context[:current_user].account_id,
-          benchmark_id: args[:benchmark_id],
           name: args[:name],
-          ref_id: args[:ref_id],
-          parent_profile_id: args[:clone_from_profile_id],
           description: args[:description],
           business_objective_id: args[:business_objective_id],
           compliance_threshold: args[:compliance_threshold]
         }
       end
+
+      def new_profile_options(args)
+        {
+          account_id: context[:current_user].account_id,
+          parent_profile_id: args[:clone_from_profile_id]
+        }
+      end
```

Comment:
The more I work between graphql and REST APIs, the more I want to consolidate the logic. I'm intentionally not doing it here mostly because the PR is so big already, but also because we may remove the graphql API completely at some point.